### PR TITLE
Remove babel dependencies + move react to peer dependencies

### PR DESCRIPTION
**Pull request by MaximeCheramy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,10 +36,12 @@
...
     "eslint-plugin-react": "7.27.0",
     "eslint-plugin-react-hooks": "4.3.0",
-    "eslint-plugin-standard": "5.0.0",
     "eslint-plugin-unused-imports": "2.0.0",
     "typescript": "4.4.4"
   },
+  "peerDependencies": {
+    "react": ">=16.0.0"
+  },
   "browserslist": [
```
Comment (MaximeCheramy) [inline package.json:45]: @CGNonofr I'm not sure this `browserslist` is used?

Comment (CGNonofr) [reply package.json:45]: It's probably not, I forgot to clean a lot of things from the projects using rollup

Comment (CGNonofr) [reply package.json:45]: Btw, all these issues are on monaco-languageclient-react as well

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@codingame/monaco-editor-react",
-  "version": "1.0.1",
+  "version": "1.0.2",
```
Comment (CGNonofr) [inline package.json:3]: What about the package-lock?

Comment (MaximeCheramy) [reply package.json:3]: fixed 8bd9465

File: package.json (hunk 1 above)
Comment (MaximeCheramy) [reply package.json]: removed

File: package.json (hunk 2 above)
Comment (CGNonofr) [reply package.json:3]: The published version is the 1.1.0, I probably forgot to increase the version here... can you switch to the 1.1.1 instead?

Comment (MaximeCheramy) [reply package.json]: https://github.com/CodinGame/monaco-editor-react/pull/2/commits/902506fb4da5cae31f873af43a1021a247c37eb3

**Review by CGNonofr — APPROVED** [review]

**Pull request merged**